log/sequencer.go

```diff
@@ -310,18 +311,26 @@ func (s Sequencer) IntegrateBatch(ctx context.Context, tree *trillian.Tree, limi
 	label := strconv.FormatInt(tree.TreeId, 10)
 
 	numLeaves := 0
-	var newLogRoot *trillian.SignedLogRoot
+	var newLogRoot *types.LogRootV1
+	var newSLR *trillian.SignedLogRoot
 	err := s.logStorage.ReadWriteTransaction(ctx, tree, func(ctx context.Context, tx storage.LogTreeTX) error {
 		stageStart := s.timeSource.Now()
 		defer seqBatches.Inc(label)
 		defer func() { seqLatency.Observe(util.SecondsSince(s.timeSource, start), label) }()
 
 		// Get the latest known root from storage
-		currentRoot, err := tx.LatestSignedLogRoot(ctx)
+		sth, err := tx.LatestSignedLogRoot(ctx)
 		if err != nil {
 			glog.Warningf("%v: Sequencer failed to get latest root: %v", tree.TreeId, err)
 			return err
 		}
+		// There is no trust boundary between the signer and the
+		// database, so we skip signature verification.
```

Comment:
We probably _should_ check it though, if only to detect corruptions etc. Maybe pop a TODO?

Reply:
done